doc/interfaces/gs_usb.rst

```diff
@@ -0,0 +1,30 @@
+.. _gs_usb:
+
+CAN driver based on WCID for Geschwister Schneider USB/CAN devices and bytewerk.org candleLight USB CAN interfaces
+==================================================================================================================
+
+Windows/Linux/Mac CAN driver based on WCID for Geschwister Schneider USB/CAN devices and candleLight USB CAN interfaces.
+
+Install: ``pip install "python-can[gs_usb]"``
+
+Usage: pass ``bus`` and ``address`` to open the device. The parameters can be got by ``pyusb``
```

Comment:
Please document the complete setup for a first time user?

Assume someone has found the python-can docs and hasn't had any direct experience with `pyusb`.

Reply:
Docs is updated